react/features/recording/components/Recording/web/StartRecordingDialog.js

```diff
@@ -19,6 +19,31 @@ import StartRecordingDialogContent from '../StartRecordingDialogContent';
  * @augments Component
  */
 class StartRecordingDialog extends AbstractStartRecordingDialog {
+
+    _onDisableStartRecording: () => boolean;
+
+    /**
+     * Disables start recording button.
+     *
+     * @returns {boolean}
+     */
+    _onDisableStartRecording() {
```

Comment:
Typically `onXXX` is an event handler. It's very confusing to read. Can you please call this `isRecordingDisabled` ?